client/shipping-label/views/history.js

```diff
@@ -0,0 +1,79 @@
+import React, { PropTypes } from 'react';
+import CalypsoI18n from 'i18n-calypso';
+import Gridicon from 'components/gridicon';
+import formatDate from 'lib/utils/format-date';
+import { translate as __ } from 'lib/mixins/i18n';
+import { sprintf } from 'sprintf-js';
+const LabelHistory = ( { labelsHistory } ) => {
+	if ( ! labelsHistory || ! labelsHistory.length ) {
+		document.getElementById( 'woocommerce-order-label-history' ).style.display = 'none';
+		return null;
+	}
+	document.getElementById( 'woocommerce-order-label-history' ).style.display = 'block';
```

Comment:
_This comment applies to both uses of `style.display =` above_

Shouldn't this be applied with a class that gets dynamically set based on the value of `labelHistory`?